src/order/bounds.lean

```diff
@@ -15,11 +15,17 @@ section preorder
 
 variables [preorder α] [preorder β] {f : α → β}
 
+/-- The set of upper bounds of a set. -/
 def upper_bounds (s : set α) : set α := { x | ∀a ∈ s, a ≤ x }
+/-- The set of lower bounds of a set. -/
 def lower_bounds (s : set α) : set α := { x | ∀a ∈ s, x ≤ a }
+/-- `a` is the least element of a set `s`. -/
```

Comment:
With just the assumption of a preorder, least elements aren't unique, so this phrasing is misleading.

Reply:
@robertylewis Tried to rephrase. Does it look better now?